openfisca_us/variables/usda/snap/income/deductions/shelter_expense.py

```diff
@@ -0,0 +1,81 @@
+from openfisca_us.model_api import *
+
+
+class snap_shelter_deduction(Variable):
+    value_type = float
+    entity = SPMUnit
+    definition_period = YEAR
+    documentation = (
+        "Excess shelter deduction for calculating SNAP benefit amount"
+    )
+    label = "SNAP shelter deduction"
+    reference = ("United States Code, Title 7, Section 2014(d)(6)",)
```

Comment:
Can we point to href/title pairs like in parameters? If not I think I'd prefer URLs.

Reply:
Unfortunately not - blocked by Core

Reply:
I see your point, but I don't think we can link to specific lines with as much precision with URLS?

Reply:
We can! https://www.law.cornell.edu/uscode/text/7/2014#d_6

Reply:
Brain fail, yeah let's use those

Reply:
Updated